### PR TITLE
Renaming base fields

### PR DESCRIPTION
**Pull request by fabricio-aguiar**
_id to pulp_id
_created to pulp_created
_last_updated to pulp_last_updated
_href to pulp_href

ref #5457
https://pulp.plan.io/issues/5457

Required PR: https://github.com/pulp/pulpcore/pull/317
Required PR: https://github.com/pulp/pulpcore-plugin/pull/137
Required PR: https://github.com/PulpQE/pulp-smash/pull/1220

### CODE REVIEW

**Comment by daviddavis** [comment]
@mdellweg this can/should be merged as the field name changes in core have been merged.

File: pip-wheel-metadata/pulp_deb.dist-info/LICENSE [hunk 1]
```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (mdellweg) [inline pip-wheel-metadata/pulp_deb.dist-info/LICENSE]: I don't think, this belongs here.

Maybe add `/pip-wheel-metadata` to `.gitignore`.

File: pip-wheel-metadata/pulp_deb.dist-info/METADATA [hunk 2]
```diff
@@ -0,0 +1,24 @@
+Metadata-Version: 2.1
```
Comment (mdellweg) [inline pip-wheel-metadata/pulp_deb.dist-info/METADATA]: This, too.

File: pip-wheel-metadata/pulp_deb.dist-info/entry_points.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+[pulpcore.plugin]
```
Comment (mdellweg) [inline pip-wheel-metadata/pulp_deb.dist-info/entry_points.txt]: Just the whole directory.

File: pip-wheel-metadata/pulp_deb.dist-info/top_level.txt [hunk 4]
```diff
@@ -0,0 +1 @@
+pulp_deb
```
Comment (mdellweg) [inline pip-wheel-metadata/pulp_deb.dist-info/top_level.txt]: Finally...

**Comment by mdellweg** [comment]
for reference: https://github.com/pulp/plugin_template/pull/125

**Comment by mdellweg** [comment]
Also please consider using:
https://github.com/pulp/plugin_template/pull/116,
https://github.com/pulp/plugin_template/pull/124 and
https://github.com/pulp/plugin_template/pull/125
for rebuilding the ci files.

**Comment by mdellweg** [comment]
@fabricio-aguiar I cherry-picked your changes to another branch, where i had the travis update already. Thank you!

File: pip-wheel-metadata/pulp_deb.dist-info/LICENSE (hunk 1 above)
Comment (fabricio-aguiar) [reply pip-wheel-metadata/pulp_deb.dist-info/LICENSE]: sorry for that, I rebased so many PRs yesterday that I forgot to add it to gitignore

Comment (mdellweg) [reply pip-wheel-metadata/pulp_deb.dist-info/LICENSE]: No problem.

**Pull request closed without merging**